tests/juliet/test_data_context_packet.py

```diff
@@ -86,7 +86,7 @@ def test_dcp_packet(simple_dcp_packet):
     assert packet.fractional_timestamp[1] == 0x10000000
     assert packet.sampling_rate == 160
     assert packet.dwell == 20018823430144 * 10 **-9#((0x00001234 << 32) + 0xFEDC0000) * 10**-9
-    assert packet.freq == -1 * 10**-9
+    assert packet.freq == 17592.186044415
```

Comment:
This test change requires an explaination:
The equation for calculating frequency is: ((word1 << 32) | word2) * (10**-9) * (2**-20))
In the Mock data word1 = 0xFFFFFFFF (4294967295) and word2 = 0xFFF00000 (4293918720)
word1 << 32 = 0xFFFFFFFF00000000 (18446744069414584320)
word1 << 32 | word2 = 0xFFFFFFFFFFF00000 (18446744073708503040)
18446744073708503040* 10^-9 *2^-20 = 17592.186044415

I don't know how this ever equaled -.000000001